### ISSUE TITLE
Server function error handler

### ISSUE THREAD

**Issue by LeonardoEichs** [nullstack/nullstack#252]
When adding Sentry's library for Node, it is recommended to add a middleware right before where the custom error handler should be, so that when an uncaught error occurs it passes through this middleware before going to the error handler.

```
import express from "express";
import * as Sentry from "@sentry/node";

// or using CommonJS
// const express = require('express');
// const Sentry = require('@sentry/node');

const app = express();

Sentry.init({ dsn: "https://examplePublicKey@o0.ingest.sentry.io/0" });

// The request handler must be the first middleware on the app
app.use(Sentry.Handlers.requestHandler());

// All controllers should live here
app.get("/", function rootHandler(req, res) {
  res.end("Hello world!");
});

// The error handler must be before any other error middleware and after all controllers
app.use(Sentry.Handlers.errorHandler());

// Optional fallthrough error handler
app.use(function onError(err, req, res, next) {
  // The error id is attached to `res.sentry` to be returned
  // and optionally displayed to the user for support.
  res.statusCode = 500;
  res.end(res.sentry + "\n");
});

app.listen(3000);
```

If I try to replicate this with Nullstack, when an **uncaught** error occurs in a **server function**, the middleware is not used because the error is treated by the server function's error handler, so it is not passing through the middleware that is before Express' error handler.

```
Sentry.init({
  dsn: context.secrets.sentryDsn,
  integrations: [
    new Sentry.Integrations.Http({ tracing: true }),
    new Tracing.Integrations.Express({ app: context.server }),
  ],
  tracesSampleRate: 1.0,
})
context.sentry = Sentry
context.server.use(Sentry.Handlers.requestHandler())
context.server.use(Sentry.Handlers.tracingHandler())
// ...
context.server.get('/hello', async (req, res) => {
  // ...
})
context.server.use(Sentry.Handlers.errorHandler()) // uncaught errors in server functions never reach here
export default context
```

**Comment by victorfernandesraton** [comment]
i search in sentry configuration for next.js and I have an idea, but I not sure if this work

https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/

you try create a custom error page and redirect server error to your page, and implement sentry in this page, it's a workaround because every API errors rely a redirect to web page and use sentry in here

**Comment by Mortaro** [comment]
We now have a context.catch interceptor that runs every error over it, both on client and server, its available on 0.18, PRs with documentation are welcome 💜😈💜
